src/applications/check-in/utils/formatters/index.js

```diff
@@ -24,4 +24,14 @@ const formatDemographicString = demographicString => {
   return demographicString;
 };
 
-export { formatPhone, formatDemographicString };
+/**
+ * @param {object} dateObject
+ */
+const formatDateObjectTo8601 = dateObject => {
+  return `${dateObject.year.value}-${String(dateObject.month.value).padStart(
```

Comment:
Could we use https://date-fns.org/v2.28.0/docs/formatISO with `{ representation: 'date' }` for this?

Reply:
Change to extractDateFromVaDateComponent

Reply:
Add comment in doc block